core/src/main/java/com/linecorp/armeria/client/ClientRequestContext.java

```diff
@@ -632,34 +634,9 @@ ClientRequestContext newDerivedContext(RequestId id, @Nullable HttpRequest req,
     HttpHeaders additionalRequestHeaders();
 
     /**
-     * Sets a header with the specified {@code name} and {@code value}. This will remove all previous values
-     * associated with the specified {@code name}.
-     * The header will be included when a {@link Client} sends an {@link HttpRequest}.
-     */
-    void setAdditionalRequestHeader(CharSequence name, Object value);
```

Comment:
How about reviving two methods - `{add,set}AdditionalRequestHeader(CharSequence, Object)` - since it's used pretty commonly?

Reply:
Yes, let me put this back. 😉